pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1506,11 +1506,11 @@ func SecretToLibvirtSecret(vmi *v1.VirtualMachineInstance, secretName string) st
 func QuantityToByte(quantity resource.Quantity) (Memory, error) {
 	memorySize, _ := quantity.AsInt64()
 	if memorySize < 0 {
-		return Memory{Unit: "B"}, fmt.Errorf("Memory size '%s' must be greater than or equal to 0", quantity.String())
+		return Memory{Unit: "b"}, fmt.Errorf("Memory size '%s' must be greater than or equal to 0", quantity.String())
```

Comment:
Just double checking, could this have implications on migrations on existing vms which were started with B?

Reply:
I really doubt it. the domain is transfered by libvirt to libvirt, no conversion is happening in our api during migrations.